### ISSUE TITLE
Wrong Zone Lock Influence Reward

### ISSUE THREAD

**Issue by Topologic** [WarEmu/WarBugs#5849]
If a zone locks and you get the tick, the influence portion of it is applied to the chapter you are physically within, as opposed to applying directly to open RvR influence all the time.

As an example, I've received three zone lock ticks while in the Dragonwake warcamp waiting for scenario pops, and each tick has applied to High Elf chapter 18 influence.

**Comment by Luth-RoR** [comment]
You shouldn't get any zone lock reward in the warcamp at all. Seems like an issue with area discovery/borders.

**Comment by Topologic** [comment]
I don't know if this has been fixed directly or indirectly since I posted the ticket back in May.  

That said, it could very well be a matter of the server thinking you're in the RvR lake when you're actually sitting in the warcamp, but would this particular explanation also imply that being in the warcamp will flag you for RvR in the same way that stepping into any RvR lake does?  I don't remember if I'd ever been flagged while sitting in the warcamp, so I don't know how useful this question would be for diagnosing the problem at hand, if it's even useful at all.
